src/workerd/api/node/zlib-util.h

```diff
@@ -69,6 +71,15 @@ enum class ZlibMode : ZlibModeValue {
   BROTLI_ENCODE
 };
 
+// Zlib OS_CODE. Needed only for test fixtures
+#if defined(__APPLE__)
+static constexpr auto ZLIB_OS_CODE = 0x13;
+#elif defined(__WIN32)
+static constexpr auto ZLIB_OS_CODE = 0xa;
+#else  // non-Apple UNIX
+static constexpr auto ZLIB_OS_CODE = 0x3;
+#endif
```

Comment:
Do we really need to export this? The test could simply mask out this field and ignore it.

Reply:
Oh that's better. I don't know why I didn't think of that